src/WebJobs.Script/Utility.cs

```diff
@@ -159,5 +160,16 @@ public static string ToLowerFirstCharacter(string input)
 
             return input;
         }
+
+        public static string RemoveUtf8ByteOrderMark(string input)
```

Comment:
Thought about doing that as this implementation also performs **significantly** better (what we had was fine for the tests). I'll put additional tests in place and replace the old implementation.